0_intro/exercises/cat/src/main.rs

```diff
@@ -1,7 +1,18 @@
+fn getstring(s : Option<String>) -> String{
+    match s{
+        Some(s) => s,
+        None => String::from(""),
+    }
+}
 pub fn cat(path: &str) -> String {
-    todo!();
+    let buffer = std::fs::read_to_string(path).expect("Something went wrong reading the file");
+    return buffer;
```

Comment:
You could just ``std::fs::read_to_string(path).expect("Something went wrong reading the file")`` without return and no ';'